test/e2e/util_test.go

```diff
@@ -39,7 +39,7 @@ import (
 
 const (
 	pollInterval = 1 * time.Second
-	pollDuration = 3 * time.Minute
+	pollDuration = 4 * time.Minute
```

Comment:
I get concerned whenever we wait too long for results since we are likely to run into a resync. Maybe our tests should set our operator's resync interval to 1hr so we don't get fooled into thinking our control loops are reasonably fast.